src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -96,16 +96,26 @@ public function without(string ...$keys): self
         return $clone;
     }
 
-    public function add(AbstractStimulusDto $stimulusDto): self
+    public function appendController(AbstractStimulusDto $stimulusDto): self
```

Comment:
IMHO we should change the type hint to `StimulusControllersDto` as it only works with that correctly. Or we should create a generic method and decide what to do based on the type.

Reply:
@weaverryan what is your preference? I like `append/prepend` better too, should I continue with it?

Reply:
I like your proposal to do something like `array|AbstractStimulusDto`... because, why not?

I'm not sure about `append/prepend`. Or, I want to try to push back on it a little bit :). The order of the Stimulus controllers should not matter. I think that is a "design problem" with how we currently fetch "Component" instances - and we should fix that a different way - probably solution (C) from here https://github.com/symfony/ux/issues/612#issuecomment-1353085729

So, assuming we fix that, is there really any reason to have a prepend vs append? That's not something you normally care about with element attributes.

Reply:
> I'm not sure about append/prepend. Or, I want to try to push back on it a little bit :). The order of the Stimulus controllers should not matter. I think that is a "design problem" with how we currently fetch "Component" instances - and we should fix that a different way - probably solution (C) from here https://github.com/symfony/ux/issues/612#issuecomment-1353085729 

I found this test [value_list_observer_tests.ts#L26-L35](https://github.com/hotwired/stimulus/blob/2ff54403a587c8b5872144008178ddfae0902657/src/tests/modules/mutation-observers/value_list_observer_tests.ts#L26-L35), which tells me that the order of tokens in the same attribute matters for Stimulus, because changing `data-controller="one"` to `data-controller="two one"`, it first disconnects `one` and after that it connects `two` and after `one` again, to ensure `two` connected first and `one` after that. Of course it's better to not rely on that in our code, but that maybe not always possible.  

For example
```ts
// foo_controller.ts
export default class extends Controller<HTMLElement> {
    override connect() {
        this.dispatch('connect')
    }
}

// hello_controller.ts
export default class extends Controller<HTMLElement> {
    onFooConnect(e: CustomEvent) {
        alert('foo connected')
    }
}
```
This will not work
```html
<div data-controller="foo hello" data-action="foo:connect->hello#onFooConnect">
```
but this will work
```html
<div data-controller="hello foo" data-action="foo:connect->hello#onFooConnect">
```

This definitely seems like an edge case to me and I can't come up with a good practical example.